plugins/coral-plugin-respect/client/style.css

```diff
@@ -0,0 +1,27 @@
+.Respect {
+    display: inline-block;
+    button {
```

Comment:
The reasons for this is, that we want to isolate the styles for this Component and don't leak out styles that might affect other components.